### ISSUE TITLE
Syntax Errors, util.py not found, cannot import name 'logsumexp' from 'autograd.scipy.misc'

### ISSUE THREAD

**Issue by Cameron-Nguyen1** [HIPS/neural-fingerprint#12]
As title says, I've encountered a whole handful of issues attempting to use this software.

After building, I had to manually place packages and libs in my miniconda3 folder. Maybe I could've specified some parameters to install it correctly? Not sure. Here are the real problems:

Attempting to import neuralfingerprint from any IDE or location within terminal outside of neuralfingerprint folder results in: 'util.py' not found. Not a problem right? Just append the path of that folder to my sys.path....

Once fixing the util.py problem, I receive many syntax errors regarding print statements. 
E.G.:
_print "Total number of weights in the network:", num_weights
          ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print("Total number of weights in the network:", num_weights)?_

Once fixing those manually, I get the next problem!

_File "/mnt/c/Users/came/Documents/neural-fingerprint-master/neural-fingerprint-master/neuralfingerprint/build_convnet.py", line 2, in <module>
    from autograd.scipy.misc import logsumexp
ImportError: cannot import name 'logsumexp' from 'autograd.scipy.misc' (/home/azn/miniconda3/lib/python3.9/site-packages/autograd/scipy/misc.py)_

I'm no longer interested in putting out fires so I figured I'd post here despite the forum being inactive for the last four years, fingers crossed y'all are still paying attention to this page!

Please let me know if there are any fixes for these issues or if y'all know what's going on here.

I believe it could just be a mismatch between Python3 and Python2, I'm not very thrilled about having to swap python3 to python2 for this so I figured it would be good to ask first.

Many thanks!

**Comment by duvenaud** [comment]
Yes, this repo was written in Python 2.  I appreciate your interest, but I think realistically that this whole package should probably be deprecated.  Let me know what equivalent modern package you end up using, and we'll add a pointer to it on the main page.
